doc/rst/tools/chapel-py/chapel-py.rst

```diff
@@ -67,7 +71,10 @@ python be installed (for many package managers this is called
 ``python3-devel``). With those constraints met, you can just run ``make
 chapel-py-venv``:
 
-This will allow you to use the python bindings from a python script run with ``$CHPL_HOME/util/config/run-in-venv-with-python-bindings.bash``, instead of ``python3``. To install the python bindings in your system python, run ``(python3 -m pip install $CHPL_HOME/tools/chapel-py)``.
+This will allow you to use the python bindings from a python script run with
+``$CHPL_HOME/util/config/run-in-venv-with-python-bindings.bash python3``,
+instead of just ``python3``. To install the python bindings in your system
+python, run ``python3 -m pip install $CHPL_HOME/tools/chapel-py``.
```

Comment:
One more thing to mention is perhaps that this prevents installing globally to your Python environment.